modules/hitlnext/src/views/client.ts

```diff
@@ -87,7 +87,7 @@ export const makeClient = (bp: { axios: AxiosInstance }): HitlClient => {
         .then(data => data.map(castHandoff)),
     assignHandoff: async id =>
       bp.axios
-        .post(`/handoffs/${id}/assign`, null, config)
+        .post(`/handoffs/${id}/assign`, { webSessionId: (window as any).__BP_VISITOR_SOCKET_ID }, config)
```

Comment:
Zero sure about this one. Any input would be appreciated :)

Reply:
webSessionId is only used in channel-web

Reply:
When you make a call to `/assign`, the module's back-end make a [call](https://github.com/botpress/botpress/blob/master/modules/hitlnext/src/backend/api.ts#L231) to the channel-web's back-end to create a new conversion. If you look at the middlewares used by this [route](https://github.com/botpress/botpress/blob/master/modules/channel-web/src/backend/api.ts#L413), you can see that it uses `assertUserInfo`.

Then if you take a look at [assertUserInfo](https://github.com/botpress/botpress/blob/master/modules/channel-web/src/backend/api.ts#L109), you can see that it fetches the `userId` with `bp.realtime.getVisitorIdFromGuestSocketId(webSessionId)`. Meaning that if you don't send a `webSessionId` with your request payload, the `userId` will not be found and send a bad request (see lines 114-116)!